test/unit/http_method/Makefile

```
@@ -49,17 +50,35 @@ MKDIR		:=	mkdir -p
 all: $(NAME)
 
 $(NAME): $(OBJS)
+	@make init
 	$(CXX) -o $@ $^
 
+# create test file and directory
+.PHONY	: init
+init: $(TEST_CASE_DIR)/no_authority_file $(TEST_CASE_DIR)/no_authority_directory
+
+$(TEST_CASE_DIR)/no_authority_file:
+	-(mkdir -p $(TEST_CASE_DIR) && touch $@ && chmod 000 $@)
+
+$(TEST_CASE_DIR)/no_authority_directory:
+	-(mkdir -p $(TEST_CASE_DIR) && mkdir -p $@ && chmod 000 $@)
+
 vpath %.cpp $(SRCS_DIR)
 $(OBJ_DIR)/%.o: %.cpp
 	@$(MKDIR) $(dir $@)
 	$(CXX) $(CXXFLAGS) $(INCLUDES) -c $< -o $@
 
 .PHONY	: clean
 clean:
+	-@clean-test
```

この部分、make cleanを実行してみるとわかるんだけど、clean-testが直で入力されて
```
make: clean-test: No such file or directory
make: [clean] Error 1 (ignored)
rm -f -r objs
```
になるから
```suggestion
clean: clean-test
```
が正しいかな

---

それだと`clean-test`で失敗してobjsが削除されないから今のままでいいと思う

```
make clean
rm -f test/no_authority_file
chmod: test/no_authority_directory: No such file or directory
make: *** [clean-test] Error 1
```

---

確かに、直で入力されてるから`make clean-test`したら動いた！